docs/operations/request-logging.md

```diff
@@ -0,0 +1,249 @@
+---
+id: request-logging
+title: Request logging
+sidebar_label: Request logging
+---
+
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+All Apache Druid services that can serve queries can also log the query requests they see.
+Request logs contain information on query metrics, including execution time and memory usage.
+You can use information in the request logs to monitor query performance, determine bottlenecks, and analyze and improve slow queries.
+
+Request logging is disabled by default.
+This topic describes how to configure Druid to log and emit query metrics.
```

Comment:
Is this query metrics? Its more of query logs with associated metrics and dimensions
@suneet-s

Reply:
The logs contain the metrics like [`sqlQuery/time`](https://druid.apache.org/docs/latest/operations/metrics.html#sql-metrics) so not sure if this is a matter of semantics. 🤷 
If it's clearer, we could change the phrasing to be something like the following:
```suggestion
This topic describes how to configure Druid to generate request logs to track query metrics.
```

Reply:
thats better